.github/workflows/develop-deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Develop Firebase functions deploy
+
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  deploy-functions:
+    runs-on: ubuntu-latest
+    environment: 
+      name: develop
+    defaults:
+      run:
+        working-directory: functions
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@v0'
+        with:
+          credentials_json: '${{ secrets.FIREBASE_SERVICE_ACCOUNT }}'
+      - name: Install dependencies
+        run: yarn global add firebase-tools && yarn install
+      - name: Deploy functions
+        run: |
+          echo "Deploying to develop..."
+          firebase deploy --only functions --project develop
+          echo "Finished deploy to develop."
```

Comment:
yeah, they're not really helpful 😆 

Previously I had them because the environment was a variable, so it might have been nice to be able to easily sanity check that the correct env was chosen, but now that they're static there's no use

Reply:
removed in 36b21deb12345aa5ef8e454b3dd8d6b1bb12570a and 53377e25e075ef03d254144d83d66b2f86f9b5b6